examples/tree_2d_dgsem/elixir_mhdmultiion_ec.jl

```diff
@@ -0,0 +1,61 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the ideal multi-ion MHD equations
+equations = IdealGlmMhdMultiIonEquations2D(gammas = (1.4, 1.667),
```

Comment:
Why do you use `1.667` here instead of `5/3`?

Reply:
For no particular reason. This elixir is only meant to test the code and does not carry any particular physical significance. If it is deemed important to change this constant, I suggest to do it in a future PR (together with the reference values) as the current value does not affect the functionality of the added code.